### PR TITLE
Set queue priority for jobs

### PR DESCRIPTION
**Pull request by sampoder**
## Summary of the problem

Flagged by Gary in #4417.

## Describe your changes

I've gone through each job and checked the priority. I've also removed them from `schedule.yml`.

### CODE REVIEW

File: app/jobs/canonical_pending_transaction_job/send_twilio_declined_message.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 module CanonicalPendingTransactionJob
   class SendTwilioDeclinedMessage < ApplicationJob
-    queue_as :critical
```
Comment (garyhtou) [inline app/jobs/canonical_pending_transaction_job/send_twilio_declined_message.rb]: is there a reason for switching from `critical` to `default`?

i think there's a high priority for these to go out ASAP; especially if someone's swiping a card in person and is wondering why it declined

Comment (sampoder) [reply app/jobs/canonical_pending_transaction_job/send_twilio_declined_message.rb]: I'm cool with it either way, I just thought this was on equal standing with a mailer

File: app/jobs/canonical_pending_transaction_job/send_twilio_declined_message.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+    queue_as :default
```
Comment (garyhtou) [inline app/jobs/canonical_pending_transaction_job/send_twilio_declined_message.rb]:
```suggestion
    queue_as :critical
```

File: app/jobs/canonical_pending_transaction_job/send_twilio_receipt_message.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 module CanonicalPendingTransactionJob
   class SendTwilioReceiptMessage < ApplicationJob
-    queue_as :critical
+    queue_as :default
```
Comment (garyhtou) [inline app/jobs/canonical_pending_transaction_job/send_twilio_receipt_message.rb]:
```suggestion
    queue_as :critical
```

**Review by garyhtou — APPROVED** [review]
great! ty for this Sam!

**Pull request merged**